MSAL/src/MSALResult.m

```diff
@@ -81,7 +82,17 @@ + (MSALResult *)resultWithTokenResult:(MSIDTokenResult *)tokenResult
     }
 
     MSIDAccount *resultAccount = tokenResult.account;
-    NSString *tenantId = [tokenResult.authority.url msidTenant];
+    NSError *claimsError = nil;
```

Comment:
Would be nice to add a unit test for this

Reply:
Done. Added a new test class to test MSALResult.